### PR TITLE
slackcat: switch to upstream vendoring

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
https://github.com/Homebrew/homebrew-core/issues/21801

### CODE REVIEW

File: Formula/slackcat.rb [hunk 1]
```diff
@@ -13,38 +11,17 @@ class Slackcat < Formula
...
-    url "https://github.com/urfave/cli.git",
-        :revision => "119bb6564841921ce6f1401e0f5d75317bdd9f4d"
-  end
-
   def install
     ENV["GOPATH"] = buildpath
-    Language::Go.stage_deps resources, buildpath/"src"
-    system "go", "build", "-o", bin/"slackcat",
+    (buildpath/"src/github.com/vektorlab/slackcat").install buildpath.children
+    cd "src/github.com/vektorlab/slackcat" do
```
Comment (DomT4) [inline Formula/slackcat.rb]: Using this style of installation you'll likely need a `prefix.install_metafiles` as the last line of this block.

Comment (commitay) [reply Formula/slackcat.rb]: Thats twice now I've forgotten about `metafiles`. 😞  

Thanks @DomT4!

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
:cat2:

File: Formula/slackcat.rb (hunk 1 above)
Comment (DomT4) [reply Formula/slackcat.rb]: No worries! It happens so often I have a GitHub quick reply for it _(not used here)_ so if you only slip twice handling quite a few of these you're not doing too badly at all 😸.

**Pull request closed without merging**